Dockerfile

```diff
@@ -1,11 +1,13 @@
-FROM golang:1.16.15@sha256:35fa3cfd4ec01a520f6986535d8f70a5eeef2d40fb8019ff626da24989bdd4f1
+FROM --platform=${BUILDPLATFORM} golang:1.19.3@sha256:d388153691a825844ebb3586dd04d1c60a2215522cc445701424205dffc8a83e
```

Comment:
I see the "pinning by digest" was added in https://github.com/cloudflare/cfssl/pull/1047 (but not mentioned). Wondering if there was a strong reason to use these

Note that when using a combination of `tag` and `digest`, the `tag` is considered "for informational purposes" and ignored. When specifying a digest, no "lookup" is performed for the tag, and the image is pulled by its digest. 

While this allows for using an immutable reference, it can also be error-prone. I'm wondering if there was a strong reason to pin it to a digest, or if a regular `:tag` would be okay for this.